drafts/developer-guide/pages/access-control/permissions/evaluate-applied-permissions.adoc

```diff
@@ -0,0 +1,148 @@
+include::ROOT:partial$variables.adoc[]
+
+= Evaluate applied permissions
```

Comment:
This document doesn't address what we discussed the last time -- permission evaluation is not specific to the admin web application roles but over other resources as well (such as modules and pages). That generalization is missing.

Reply:
I've pointed out my solution [here](https://docs.cortezaproject.org/corteza-docs/2022.3/integrator-guide/security-model/index.html) -- the `NOTE` block which you could consider here also